### DISCUSSION TITLE
Arguments?

### DISCUSSION THREAD

**Discussion by scottonanski** [lllyasviel/stable-diffusion-webui-forge#682] [Q&A]
I'm getting the following message at launch;

```
Hint: your device supports --pin-shared-memory for potential speed improvements.
Hint: your device supports --cuda-malloc for potential speed improvements.
Hint: your device supports --cuda-stream for potential speed improvements.
CUDA Stream Activated:  False
```

Now, the docs say that you can pass in arguments, but it doesn't tell you HOW to pass those arguments in.

Can anyone help? I've tried searching for it, but if you don't know what to ask then you can't find what you need to know. 

Thanks.

**Comment by Tolga077** [comment]
open your webui-user.bat with any text editor and add your desired commands to COMMANDLINE_ARGS=   part. This is my commands.
set COMMANDLINE_ARGS=    --xformers --force-enable-xformers --xformers-flash-attention  --unet-in-fp8-e5m2 --clip-in-fp8-e5m2
